metric/builder.go

```diff
@@ -0,0 +1,55 @@
+package metric
+
+import (
+	"time"
+
+	"github.com/influxdata/telegraf"
+)
+
+type TimeFunc func() time.Time
+
+type Builder struct {
+	TimeFunc
+	TimePrecision time.Duration
+
+	*metric
+}
+
+func NewBuilder() *Builder {
```

Comment:
Really don't like metric builders. They are not so complicated that you need a builder to return a metric. This also isn't used anywhere and should be removed.

Reply:
Yeah, this isn't a part of the plugin. Likely a file from an older version that slipped through a merge/rebase/some git weirdness. I'll remove.